src/modules/map/components/footer/MapFooterAttributionList.vue

```diff
@@ -6,6 +6,7 @@
                 v-if="source.hasDataDisclaimer"
                 :source-name="source.name"
                 :complete-disclaimer-on-click="!source.url"
+                :is-external-data-local="source.isExternalDataLocal"
```

Comment:
you should also rename your prop declaration to `:is-local-file` (in ThirdPartyDisclaimer the props is now called `isLocalFile`)